src/katsdpcontroller/scheduler.py

```diff
@@ -1606,6 +1606,16 @@ def __init__(self, name):
         self.taskinfo.command.shell = False
         self.physical_factory = PhysicalTask
 
+    @property
+    def numa_nodes(self) -> float:
+        return self._numa_nodes
+
+    @numa_nodes.setter
+    def numa_nodes(self, value: float) -> None:
+        if not 0.0 <= value <= 1.0:
+            raise ValueError("numa_nodes must be between 0 and 1")
```

Comment:
I'm not even sure this question is warranted, so apologies for making you answer it. 
* Does this error message accurately describe the range of valid/required values for `numa_nodes`?
* i.e. Does it provide enough information that it can be a fractional/float value?

An error message that reads "... must be between 0.0 and 1.0" looks a bit odd
* I guess anyone using altering this can see the type-hints in this base class, and
* That should be enough ?

Let me know what your thinking was/is. Please and thank you.

Reply:
The docstring says
```
    numa_nodes : float
        Minimum fraction of a NUMA node's cores to allocate. This should only
        be set if `cores` is, and must not be set higher than 1.0.
```
which seems pretty clear that it's a fraction.